src/prisma/prisma.service.ts

```diff
@@ -12,4 +12,22 @@ export class PrismaService extends PrismaClient implements OnModuleInit {
       await app.close();
     });
   }
+
+  async cleanDatabase() {
```

Comment:
Add cleanDatabase function so that tests can manually clean db whenever neccessary

Reply:
Add test